src/dynamic-import/trailing-comma-second.case

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2021 V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: ImportCall trailing comma following second parameter
+template: syntax/valid
+info: |
+  ImportCall :
+      import( AssignmentExpression[+In, ?Yield, ?Await] ,opt )
+      import( AssignmentExpression[+In, ?Yield, ?Await] , AssignmentExpression[+In, ?Yield, ?Await] ,opt )
+features: [import-assertions]
+---*/
+
+//- import
+import('', '')
```

Comment:
> Was the intent here was to have a trailing comma against the second param
> like this?

Yes, that was the intent. Thanks for pointing this out!

> Maybe it would be good to have a .case for both `import('', '')` and
> `import('', '',)` to validate that both are now syntactically valid.

Following the correction you spotted, I think we're good on from a coverage
standpoint. This patch's hand-written tests verify the former syntax, and
they're a little stronger than these procedurally-generated tests because they
also verify the semantics of those productions.